source/wp-content/themes/wporg-showcase-2022/patterns/secondary-search-bar.php

```diff
@@ -0,0 +1,10 @@
+<?php
+/**
+ * Title: Secondary Search Bar
+ * Slug: wporg-showcase-2022/secondary-search-bar
+ * Categories: wporg
+ */
+
+?>
+
+<!-- wp:search {"showLabel":false,"placeholder":"<?php esc_attr_e( 'Search sites...', 'wporg' ); ?>","width":100,"widthUnit":"%","buttonText":"Search","buttonPosition":"button-inside","buttonUseIcon":true,"className":"is-style-secondary-search-control"} /-->
```

Comment:
`"showLabel":false` hides the label visually, but it's still there in the content (using the default label, "Search"), so the feedback about [the overly verbose placeholder from here](https://wordpress.slack.com/archives/C02RP4X03/p1663279430556099) still applies.

Reply:
I should have verified. Thanks.